### DISCUSSION TITLE
how I can set the amounts with CreateTenderDeclarationTransactionClientRequest

### DISCUSSION THREAD

**Discussion by paryken** [microsoft/Dynamics365Commerce.Solutions#272] [Q&A]
When I generate a transaction using the POSAPI's CreateTenderDeclarationTransactionClientRequest, I am unable to update the values for 'Tendered' and 'Amount in transaction currency'. Even when I enter a number into columns containing the word 'amount', it results in '0'. Could you please advise how I can set the amounts correctly?
![画像 (3)](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/110579903/713ab42e-0eb1-4799-b16b-8264c10313d2)
![画像 (2)](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/110579903/2e716227-dc43-4dd9-ac24-3864d0387445)

**Comment by zhangguanghuib** [comment]
Can you check this code   https://github.com/zhangguanghuib/NewCommerceSDK/blob/main/KnowledgeArticles(small)-Development/001%20How%20to%20call%20CreateTenderDeclarationTransactionClientRequest_Response.md
